src/organisation/containers/update-pba-check/update-pba-numbers-check.component.html

```diff
@@ -0,0 +1,50 @@
+<app-hmcts-main-wrapper [backLink]="'/organisation/update-pba-numbers'" [title]="title">
+  <table class="govuk-table">
+    <tbody class="govuk-table__body">
+
+      <!-- Accounts to be added -->
+      <tr class="govuk-table__row" *ngIf="organisation.pendingAddPaymentAccount.length">
+        <th class="govuk-table__header">Accounts to be added</th>
+        <td class="govuk-table__cell">
+          <div *ngFor="let pba of organisation.pendingAddPaymentAccount">
+            {{pba}}
+          </div>
+        </td>
+        <td class="govuk-table__cell">
+          <a class="govuk-link fright" routerLink="/organisation/update-pba-numbers"
+            id="change-pba-account-numbers__link" aria-label="Change PBA account numbers">Change</a>
+        </td>
+      </tr>
+
+      <!-- Accounts to be remove -->
+      <tr class="govuk-table__row" *ngIf="organisation.pendingRemovePaymentAccount.length > 0">
+        <th class="govuk-table__header" scope="row">Accounts to be removed</th>
+        <td class="govuk-table__cell">
+          <div *ngFor="let pba of organisation.pendingRemovePaymentAccount">
+            {{pba}}
+          </div>
+        </td>
+        <td class="govuk-table__cell">
+          <a class="govuk-link fright" routerLink="/organisation/update-pba-numbers"
+            id="change-pba-account-numbers__link" aria-label="Change PBA account numbers">Change</a>
```

Comment:
UX: Same as previous comment.

Reply:
Resolving comment since it's been confirmed the UX team were happy with "PBA account numbers".